File: loisir/app/Models/Photo.php
<?php

/**
 * Created by Reliese Model.
 */

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

/**
 * Class Photo
 * 
 * @property int $id_photo
 * @property string $nom
 * @property int $id_loisir
 * 
 * @property Loisir $loisir
 *
 * @package App\Models
 */
class Photo extends Model
{
	protected $table = 'photos';
	protected $primaryKey = 'id_photo';
	public $timestamps = false;

	protected $casts = [
		'id_loisir' => 'int'
	];

	protected $fillable = [
		'nom',
		'id_loisir'
	];

	public function loisir()
	{
		return $this->belongsTo(Loisir::class, 'id_loisir');
	}
}

File: LoisirsVF/lib/model/Photo.dart
class Photo{
  String id_photo,nom;

  Photo({
    this.id_photo,
    this.nom,
  });
  factory Photo.fromJson(Map<String, dynamic> json) {
    return Photo(
      id_photo: json['id_photo'] as String,
      nom: json['nom'] as String,
    );
  }
}
